scriptmodules/emulators/hypseus.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hypseus"
+rp_module_desc="Hypseus Singe - Laserdisc Emulator"
+rp_module_help="ROM Extension: .daphne\n\nCopy your Daphne roms to $romdir/daphne"
+rp_module_licence="GPL3 https://raw.githubusercontent.com/DirtBagXon/hypseus-singe/master/LICENSE"
+rp_module_repo="git https://github.com/DirtBagXon/hypseus-singe.git RetroPie"
+rp_module_section="exp"
+rp_module_flags="sdl2"
+
+function depends_hypseus() {
+    getDepends libvorbis-dev libogg-dev zlib1g-dev libmpeg2-4-dev libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev cmake
+}
+
+function sources_hypseus() {
+    gitPullOrClone
+}
+
+function build_hypseus() {
+    local params=()
+    mkdir build
+    cd build
+    cmake ../src
+    make
+    cp hypseus ../hypseus.bin
+    md_ret_require="hypseus"
+}
+
+function install_hypseus() {
+    md_ret_files=(
+        'sound'
+        'pics'
+        'fonts'
+        'hypseus.bin'
+        'LICENSE'
+    )
+}
+
+function configure_hypseus() {
+    mkRomDir "daphne"
+    mkRomDir "daphne/roms"
+
+    addEmulator 0 "$md_id" "daphne" "$md_inst/hypseus.sh %ROM%"
+    addSystem "daphne"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    if [[ ! -f "$md_conf_root/daphne/hypinput.ini" ]]; then
+        cp -v "$md_data/hypinput.ini" "$md_conf_root/daphne/hypinput.ini"
+    fi
```

Comment:
This looks like a job for `copyDefaultConfig` 

https://retropie.org.uk/api/helpers_8sh.html#a49d0ead30005ce0d98d91e09e8a3cd2c